### ISSUE TITLE
ADSignInLogsGraph‐Analyzer not parsing

### ISSUE THREAD

**Issue by DoubtfulTurnip** [evild3ad/Microsoft-Analyzer-Suite#2]
I may have done something incorrectly so please feel free to correct me. Here are the steps I took to reproduce the issue.

- Used Azure Sign-in Logs via Graph API extractor

https://microsoft-365-extractor-suite.readthedocs.io/en/latest/functionality/AzureSignInLogsGraph.html
https://github.com/invictus-ir/Microsoft-Extractor-Suite

- The options don't appear to output a combined JSON. With no options specified I am provided a file named "YYYYMMDDHHMMSS-SignInLogsGraph.json". I have also tried using the option -MergeOutput but get the same result. The readme doesn't seem to quite match up with the output either.

- Selecting the file using the ADSignInLogsGraph‐Analyzer on this file results in lots of errors. I have to choose all files option in the browser window due to the lack of combined log output.

```
Cannot find an overload for "ToString" and the argument count: "1".
At C:\PATHTOMAS\ADSignInLogsGraph-Analyzer.ps1:356 char:5
+     $Line = [PSCustomObject]@{
+     ~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [], MethodException
    + FullyQualifiedErrorId : MethodCountCouldNotFindBest
```

My guess is I have missed a step or running this on the wrong extractor, but any help will be greatly appreciated.

**Comment by evild3ad** [comment]
ADSignInLogsGraph‐Analyzer needs to be fixed to support the output of Microsoft-Extractor-Suite v2.0.3.
There were multiple code improvements. I will see if I can upload a quick fix soon.

Please give UAL-Analyzer a try in the meantime. Possibly you can work with the UserLoggedIn and UserLoginFailed operations, but it is Single User Audit only.

**Comment by DoubtfulTurnip** [comment]
That's awesome thanks for the quick response

**Comment by evild3ad** [comment]
Sorry, but fixing / supporting v2.0.3 makes no sense, because a lot of properties are missing compared to Graph Beta. Please try Microsoft-Extractor-Suite v1.3.2. I have reported this issue already to Joey@Invictus. The Graph Beta output is much better.

**Comment by evild3ad** [comment]
Microsoft-Extractor-Suite v2.0.4 will fix this issue and return to the Graph Beta. Tested a pre-release successfully with my DEV tenant.

**Comment by evild3ad** [comment]
Microsoft-Extractor-Suite v2.1.0 is out and I released ADSignInLogsGraph-Analyzer v0.1.1. I successfuly tested it with my DEV tenant. Please let me know if everything is working for you now!
